docker/simplified_app.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+source /bd_build/buildconfig
+set -x
+
+repo="$1"
+version="$2"
+
+apt-get update && $minimal_apt_get_install python-dev \
+  python2.7 \
+  python-nose \
+  python-setuptools \
+  gcc \
+  git \
+  libpcre3 \
+  libpcre3-dev \
+  libffi-dev \
+  libjpeg-dev \
+  nodejs \
+  npm \
+  libpq-dev \
+  libxml2-dev \
+  libltdl-dev \
+  libxmlsec1 libxmlsec1-openssl libxslt1.1 libxslt-dev
+
+# Build `xmlsec1` locally to avoid dependency conflict
+# between `libssl-dev` and `libssl1.0-dev`.
+(
+  XMLSEC_VERSION="1.2.30"
+  cd /tmp && \
+  curl -L -O "http://www.aleksey.com/xmlsec/download/xmlsec1-${XMLSEC_VERSION}.tar.gz" && \
+  tar xfz "xmlsec1-${XMLSEC_VERSION}.tar.gz" && \
+  cd "xmlsec1-${XMLSEC_VERSION}" && \
+  ./configure && make && make install
+)
+rm -rf "/tmp/xmlsec1-${XMLSEC_VERSION}"
+
+# Create a user.
+useradd -ms /bin/bash -U simplified
+
+# Get the proper version of the codebase.
+mkdir /var/www && cd /var/www
+git clone https://github.com/${repo}.git circulation
```

Comment:
@jonathangreen mentioned that he'd like to see this line removed so that the Docker image is built from the same repo that contains it. I think this can wait for a follow-up branch once we verify that Docker builds still work.